sycl/CMakeLists.txt

```diff
@@ -12,8 +12,8 @@ option(SYCL_ADD_DEV_VERSION_POSTFIX "Adds -V postfix to version string" ON)
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 include(AddSYCLExecutable)
 
-set(SYCL_MAJOR_VERSION 3)
-set(SYCL_MINOR_VERSION 1)
+set(SYCL_MAJOR_VERSION 4)
```

Comment:
Shouldn't this change done in scope of 30b8acc?

Reply:
No, according to [this guide](https://github.com/intel/llvm/blob/sycl/sycl/doc/ABIPolicyGuide.md) developer only changes dev version when breaking the ABI.